### ISSUE TITLE
Unable to create tensor, you should probably activate padding with 'padding=True' to have batched tensors with the same length."

### ISSUE THREAD

**Issue by neptune-T** [ParaN3xus/typress#2]
I can open this webpage normally, and can copy photos to display, but when inserting photos, it prompts "Unable to create tensor, you should probably activate padding with 'padding=True' to have batched tensors with the same length."

**Comment by ParaN3xus** [comment]
workaround: downgrade numpy to 1.26.0.
pending fix

**Comment by neptune-T** [comment]
After I made an error in the environment configuration, I modified the environment configuration to work properly like this:
aiohappyeyeballs   2.3.4       Happy Eyeballs for asyncio
aiohttp            3.10.0      Async http client/server framework (asyncio)
aiosignal          1.3.1       aiosignal: a list of registered asynchronous callbacks
asttokens          2.4.1       Annotate AST trees with source code positions
attrs              23.2.0      Classes Without Boilerplate
blinker            1.8.2       Fast, simple object-to-object and broadcast signaling
certifi            2024.7.4    Python package for providing Mozilla's CA Bundle.
charset-normalizer 3.3.2       The Real First Universal Charset Detector. Open, modern and actively maintained alterna...
click              8.1.7       Composable command line interface toolkit
colorama           0.4.6       Cross-platform colored terminal text.
comm               0.2.2       Jupyter Python Comm implementation, for usage in ipykernel, xeus-python etc.
datasets           2.14.4      HuggingFace community-driven open-source library of datasets
debugpy            1.8.2       An implementation of the Debug Adapter Protocol for Python
decorator          5.1.1       Decorators for Humans
dill               0.3.7       serialize all of Python
evaluate           0.4.2       HuggingFace community-driven open-source library of evaluation
executing          2.0.1       Get the currently executing AST node of a frame, and other information
filelock           3.15.4      A platform independent file lock.
flask              3.0.3       A simple framework for building complex web applications.
flask-cors         4.0.1       A Flask extension adding a decorator for CORS support
frozenlist         1.4.1       A list-like structure which implements collections.abc.MutableSequence
fsspec             2024.6.1    File-system specification
huggingface-hub    0.24.5      Client library to download and publish models, datasets and other repos on the huggingf...  
idna               3.7         Internationalized Domain Names in Applications (IDNA)
ipykernel          6.29.5      IPython Kernel for Jupyter


**ipython            8.26.0      IPython: Productive Interactive Computing**


itsdangerous       2.2.0       Safely pass data to untrusted environments and back.
jedi               0.19.1      An autocompletion tool for Python that can be used for text editors.
jinja2             3.1.4       A very fast and expressive template engine.
jiwer              3.0.4       Evaluate your speech-to-text system with similarity measures such as word error rate (WER)  
jupyter-client     8.6.2       Jupyter protocol implementation and client libraries
jupyter-core       5.7.2       Jupyter core package. A base package on which Jupyter projects rely.
markupsafe         2.1.5       Safely add untrusted strings to HTML/XML markup.
matplotlib-inline  0.1.7       Inline Matplotlib backend for Jupyter
mpmath             1.3.0       Python library for arbitrary-precision floating-point arithmetic
multidict          6.0.5       multidict implementation
multiprocess       0.70.15     better multiprocessing and multithreading in Python
nest-asyncio       1.6.0       Patch asyncio to allow nested event loops
networkx           3.3         Python package for creating and manipulating graphs and networks


**numpy              1.26.0      Fundamental package for array computing in Python**


opencv-python      4.10.0.84   Wrapper package for OpenCV python bindings.
packaging          24.1        Core utilities for Python packages


**pandas             2.2.2       Powerful data structures for data analysis, time series, and statistics**


parso              0.8.4       A Python Parser
pillow             10.4.0      Python Imaging Library (Fork)
platformdirs       4.2.2       A small Python package for determining appropriate platform-specific dirs, e.g. a `user...  
prompt-toolkit     3.0.47      Library for building powerful interactive command lines in Python
psutil             6.0.0       Cross-platform lib for process and system monitoring in Python.
pure-eval          0.2.3       Safely evaluate AST nodes without side effects
pyarrow            17.0.0      Python library for Apache Arrow
pygments           2.18.0      Pygments is a syntax highlighting package written in Python.
python-dateutil    2.9.0.post0 Extensions to the standard Python datetime module
python-dotenv      1.0.1       Read key-value pairs from a .env file and set them as environment variables
pytz               2024.1      World timezone definitions, modern and historical
pywin32            306         Python for Window Extensions
pyyaml             6.0.1       YAML parser and emitter for Python
pyzmq              26.0.3      Python bindings for 0MQ
rapidfuzz          3.9.5       rapid fuzzy string matching
regex              2024.7.24   Alternative regular expression module, to replace re.
requests           2.32.3      Python HTTP for Humans.
safetensors        0.4.3      
six                1.16.0      Python 2 and 3 compatibility utilities
stack-data         0.6.3       Extract data from python stack frames and tracebacks for informative displays
sympy              1.13.1      Computer algebra system (CAS) in Python
tokenizers         0.19.1  

   
**torch              2.4.0       Tensors and Dynamic neural networks in Python with strong GPU 
acceleration**


tornado            6.4.1       Tornado is a Python web framework and asynchronous networking library, originally devel...  
tqdm               4.66.4      Fast, Extensible Progress Meter
traitlets          5.14.3      Traitlets Python configuration system


**transformers       4.43.3      State-of-the-art Machine Learning for JAX, PyTorch and TensorFlow**


typing-extensions  4.12.2      Backported and Experimental Type Hints for Python 3.8+
typst              0.11.1      Python binding to typst
tzdata             2024.1      Provider of IANA time zone data
urllib3            2.2.2       HTTP library with thread-safe connection pooling, file post, and more.
wcwidth            0.2.13      Measures the displayed width of unicode strings in a terminal
werkzeug           3.0.3       The comprehensive WSGI web application library.
xxhash             3.4.1       Python binding for xxHash
yarl               1.9.4       Yet another URL library
